# About [excep3_pt2.cpp](https://github.com/C0DER11101/CPPNotesAndPrograms/blob/master/ExceptionHandling/excep3_pt2.cpp)

excep3_pt2.cpp shows us that it's possible that arguments of several **catch** statements match the type of an exception. In such cases, the first handler that matches the exception type is executed.

So, that's exactly what happens in excep3_pt2.cpp .

---
